src/foam/comics/v2/DAOSummaryView.js

```diff
@@ -111,6 +112,18 @@ foam.CLASS({
   actions: [
     {
       name: 'edit',
+      isEnabled: function(config, data) {
+        if ( config.CRUDEnabledActionsAuth && config.CRUDEnabledActionsAuth.isEnabled ) {
+          try {
+            let permissionString = config.CRUDEnabledActionsAuth.enabledActionsAuth.permissionFactory(foam.nanos.ruler.Operations.UPDATE, data);
+            
+            return this.auth.check(null, permissionString).then(isAuthorized => isAuthorized);
```

Comment:
What does the "sAuthorized => isAuthorized" do?

Reply:
The resolved promise from `this.auth.check` returns a boolean since it is basically just checking if the user passes the permission check or not. So named the **boolean isAuthorized**. 

For the other half, I just used the ES6 arrow function for simplicity and I thought it looked cleaner, it means to just simply **return isAuthorized**. I can change it to standard JS if you want to remove ES6 code if its complicated.

Reply:
Alternative would just be
[.then(function(isAuthorized){
return isAuthorized;
});](url)

Reply:
And just doing:
  return this.auth.check(null, permissionString);
Doesn't work?